changelog/814.breaking.rst

```diff
@@ -1 +1 @@
-Rename :class:`ApplicationCommandInteractionDataResolved` to :class:`InteractionDataResolved`, and :meth:`InteractionDataResolved.get` to :meth:`~InteractionDataResolved.get_by_id`.
+Rename :meth:`InteractionDataResolved.get` to :meth:`~InteractionDataResolved.get_by_id`.
```

Comment:
given that ApplicationCommandInteractionDataResolved is no longer documented there should defintely be a changelog entry for that too